CI failing with ipyvue FileNotFoundError

Last successful log sample: https://github.com/spacetelescope/jdaviz/runs/5661939687?check_suite_focus=true

Failed log sample: https://github.com/spacetelescope/jdaviz/runs/5683120647?check_suite_focus=true (This job used to succeed but now fails when rerun)

First noticed around 2022-03-24 4:34 PM (US Eastern). No new upstream released detected in `pip freeze`.

```
py39-test-alldeps-cov run-test: commands[1] | pytest --pyargs jdaviz /home/runner/work/jdaviz/jdaviz/docs --cov jdaviz --cov-config=/home/runner/work/jdaviz/jdaviz/setup.cfg --remote-data
ImportError while loading conftest '/home/runner/work/jdaviz/jdaviz/conftest.py'.
../../conftest.py:1: in <module>
    from jdaviz import __version__
../../.tox/py39-test-alldeps-cov/lib/python3.9/site-packages/jdaviz/__init__.py:10: in <module>
    from jdaviz.app import *  # noqa
../../.tox/py39-test-alldeps-cov/lib/python3.9/site-packages/jdaviz/app.py:56: in <module>
    ipyvue.register_component_from_file(None, 'j-tooltip',
../../.tox/py39-test-alldeps-cov/lib/python3.9/site-packages/ipyvue/VueComponentRegistry.py:40: in register_component_from_file
    with open(file_name) as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/jdaviz/jdaviz/.tox/py39-test-alldeps-cov/lib/python3.9/site-packages/jdaviz/components/tooltip.vue'
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -61,8 +61,11 @@ all =
 [options.package_data]
 jdaviz =
     data/*
+    data/*/*
     *.vue
-    jdaviz/configs/imviz/imviz.yaml
+    components/*.vue
+    configs/*/*/*/*.vue
+    configs/*/*.yaml
 
 [options.entry_points]
 console_scripts =

